virttest/libvirt_vm.py

```diff
@@ -2570,6 +2570,25 @@ def vcpuinfo(self):
                 vcpuinfo_list.append(vcpuinfo_dict)
         return vcpuinfo_list
 
+    def domfsinfo(self):
+        """
+        Return a dict's list include domain mounted filesystem information
+        via virsh command
+        """
+        result = virsh.domfsinfo(self.name,
+                                 uri=self.connect_uri).stdout.strip()
```

Comment:
What about the command failure result? As you also do not deal with the failures in virsh.py

Reply:
I think the exceptions can be covered by function "command" in virsh.py, which will help raise exception if cmd failed to execute. Pls correct me if I'm wrong. thanks

Reply:
ignore_status = dargs.get('ignore_status', True) default value is True, which means will ignore status, if you do not set it will not raise error, please check command function in virsh.py

Reply:
I think you are right, the error is ignored by default. So how about change code as below, by setting  ignore_status=False?  in the func 
```
result = virsh.domfsinfo(self.name, ignore_status=False
+                                    uri=self.connect_uri).stdout.strip()
```

Or we pass ignore_status=False in dargs when invoking func

```
        command_dargs = {"readonly": readonly_mode, "quiet_mode": quiet_mode,
                         "debug": True, "ignore_status": False}
        result = virsh.domfsinfo(vm_name, **command_dargs)

```

Which one you think it's better? I checked other func in virsh.py, seemingly most commands are implemented as below, without setting ignore status implicitly 
```
def suspend(name, **dargs):
    return command("suspend %s" % (name), **dargs) 
```

Reply:
it is ok